include/fastgltf/parser.hpp

```diff
@@ -174,11 +174,16 @@ namespace fastgltf {
         // See https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_unlit/README.md
         KHR_materials_unlit = 1 << 17,
 
-		// See https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_anisotropy/README.md
-	    KHR_materials_anisotropy = 1 << 18,
+        // See https://github.com/KhronosGroup/glTF/blob/main/extensions/2.0/Khronos/KHR_materials_anisotropy/README.md
+        KHR_materials_anisotropy = 1 << 18,
 
         // See https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Vendor/EXT_mesh_gpu_instancing/README.md
         EXT_mesh_gpu_instancing = 1 << 19,
+
+#if FASTGLTF_ENABLE_DEPRECATED_EXT
+        // See https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Archived/KHR_materials_pbrSpecularGlossiness/README.md
+        KHR_materials_pbrSpecularGlossiness = 1 << 20,
```

Comment:
I guess it could also be useful to mark this [[deprecated]], which is allowed since C++17:
```cpp
KHR_materials_pbrSpecularGlossiness [[deprecated]] = 1 << 20
```

Reply:
It's not a deprecated feature of the fastgltf code but of the glTF format.
I wonder why we would tell the user he is using deprecated code (since he is not)

Reply:
I was thinking of using it as an indicator about the state of the glTF extension and not the actual code being deprecated. But sure, was only a thought anyway.